Sources/NTPClient/NTPResponse.swift

```diff
@@ -97,8 +97,16 @@ extension NTPResponse {
             throw NTPError.versionNotSupported
         }
 
+        // Catch _NTPError and convert to NTPError
+        let offset: TimeInterval
+        do {
+            offset = try p.getOffset(clientReceiveTime: clientReceiveTime)
+        } catch let e as _NTPError where e == .arithmeticOverflow {
+            throw NTPError.arithmeticOverflow
```

Comment:
What's the benefit of the catch/rethrow dance

Reply:
we're translating the error form the internal `NTP` package to something that is consumable in the public API by consumers of the `NTPClient` package. this allows us to expose an arithmeticOverfllow without exposing details of the internal, underlying package

so its a throw type conversion rather than a re-throw